### ISSUE TITLE
Wrong starting date in data when getting in Direct DB Connection mode

### ISSUE THREAD

**Issue by slako** [grafana/grafana-zabbix#1168]
When I getting data using Direct DB Connection mode, first data point is missed.
For example, I asking metrics trend data with date range starting from 2012/01/31 00:00:00, item have 5m interval.
Data inspection shows first data point is 2012/01/31 00:05:00. There is not such issue when I turn off Direct DB mode to use API, in this way I got first data point from 2012/01/31 00:00:00

Expected behavior: first data point date should be same, as in API mode

Grafana 7.4.3
Zabbix 4.0.25
Grafana-Zabbix Plugin 4.1.2

**Comment by alexanderzobnin** [comment]
Yes, it seems, range from/to timestamps should be included in the filter:

https://github.com/alexanderzobnin/grafana-zabbix/blob/29a8bde236bd8bc059a2b649e2bb55440c827bd4/src/datasource-zabbix/zabbix/connectors/sql/mysql.js#L10
